aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -541,35 +559,44 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
         uint128 recipientBalance = balance[token][recipient];
         require(recipientBalance > 0, "The recipient token balance = 0");
 
-        bytes memory args = bytes(
-            string.concat(
-                '{"receiver_id": "',
-                auroraEngineAccountIdOnNear,
-                '", "amount": "',
-                Strings.toString(recipientBalance),
-                '", "msg": "',
-                UtilsFastBridge.addressToString(recipient),
-                '"}'
-            )
-        );
         balance[token][recipient] -= recipientBalance;
 
+        string memory msgStr = UtilsFastBridge.addressToString(recipient);
+        if (UtilsFastBridge.isStrEqual(token, auroraEngineAccountIdOnNear)) {
+            string memory sixtyFourZeros;
+            for (uint8 i = 0; i < 64; i++) {
```

Comment:
This increase the gas consumption, let's try to use a const for it:
`string constant sixtyFourZeros = "0000000000000000000000000000000000000000000000000000000000000000";`, or try to find an more optimised solution if needed.

Reply:
https://github.com/aurora-is-near/fast-bridge-protocol/pull/113/commits/d099a07c443e096285172fa366e1f09539d73e41